language/stdlib/modules/validator_set.mvir

```diff
@@ -1,75 +1,29 @@
 module ValidatorSet {
-    resource T {
-        // size of the validator set. Currently serialization format will enforce the lexicographic ordering by field
```

Comment:
I believe this may not be true any longer which could make the comment "obsolete". @bob-wilson may confirm here.

Reply:
I'm still working on rebasing my change in https://github.com/libra/libra/pull/773, so it's not *yet* obsolete

Reply:
This comment is getting deleted, and it makes me very happy that it will soon be obselete!